st2client/st2client/commands/action.py

```diff
@@ -282,31 +282,24 @@ def __init__(self, resource, *args, **kwargs):
         super(ActionDeleteCommand, self).__init__(resource, *args, **kwargs)
 
         self.parser.add_argument(
-            "-f",
-            "--force",
+            "-rf",
+            "--remove-files",
             action="store_true",
-            dest="force",
-            help="Auto yes flag to delete action files from disk.",
+            dest="remove_files",
+            help="Remove action files from disk.",
         )
 
     @add_auth_token_to_kwargs_from_cli
     def run(self, args, **kwargs):
         resource_id = getattr(args, self.pk_argument_name, None)
         instance = self.get_resource(resource_id, **kwargs)
-        msg = (
-            'Resource with id "%s" has been successfully deleted from database and disk.'
-            % (resource_id)
-        )
-        user_input = ""
-        if not args.force:
-            user_input = input(
-                "The resource files on disk will be deleted. Do you want to continue? (y/n): "
-            )
-        if args.force or user_input.lower() == "y" or user_input.lower() == "yes":
-            self.manager.delete(instance, **kwargs)
-            print(msg)
+        if args.remove_files:
+            remove_files = True
         else:
-            print("Action is not deleted.")
+            remove_files = False
```

Comment:
Since the remove_files arg is already defined as boolean above, you can change this if/else block to `remove_files = args.remove_files`.